application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/spring/ECChronos.java

```diff
@@ -54,12 +55,13 @@ public ECChronos(
             final DefaultRepairConfigurationProvider defaultRepairConfigurationProvider,
             final EccNodesSync eccNodesSync,
             final RepairHistoryService repairHistoryService,
-            final RepairFaultReporter repairFaultReporter
+            final RepairFaultReporter repairFaultReporter,
+            final StatementDecorator statementDecorator
```

Comment:
Where are you creating this? From what I see on sidecar we create this in BeanConfigurator based on the configuration, but from what I see on this change it's never created, so does it mean that it fails when it start the application?

Reply:
which object/bean StatementDecorator  or  ECChronos ? will we not create ECChronos bean in BeanConfigurator? I leave it on purpose, when we will create Ecchronos bean we can inject Statement Decorator as well as bean.